reports/src/main/java/com/toggl/reports/ui/composables/AdvancedReportsOnWeb.kt

```diff
@@ -0,0 +1,68 @@
+package com.toggl.reports.ui.composables
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.ConstraintLayout
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.imageResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.toggl.common.feature.compose.theme.grid_2
+import com.toggl.reports.R
+import com.toggl.reports.domain.ReportsAction
+
+@Composable
+fun AdvancedReportsOnWeb(dispatcher: (ReportsAction) -> Unit) {
+
+    ConstraintLayout(Modifier.fillMaxWidth()) {
+
+        val (header, text, callToAction, image) = createRefs()
+
+        GroupHeader(
+            text = stringResource(R.string.advanced_reports),
+            modifier = Modifier.padding(top = grid_2) then Modifier.constrainAs(header) {
```

Comment:
Why you don't simply chain these modifiers?
```suggestion
            modifier = Modifier.padding(top = grid_2).constrainAs(header) {
```